kouta-backend/src/main/scala/fi/oph/kouta/service/KoulutusService.scala

```diff
@@ -103,24 +112,29 @@ class KoulutusService(
               case amkMetadata: AmmattikorkeakouluKoulutusMetadata =>
                 Some(amkMetadata.copy(isMuokkaajaOphVirkailija = Some(isOphVirkailija)))
               case m: AmmOpeErityisopeJaOpoKoulutusMetadata =>
-                val opintojenLaajuusKoodiUri =
-                  if (m.opintojenLaajuusKoodiUri.isDefined) m.opintojenLaajuusKoodiUri
-                  else
-                    Some(koodistoClient.getKoodiUriWithLatestVersion("opintojenlaajuus_60"))
-                val koulutusalaKoodiUrit =
-                  if (m.koulutusalaKoodiUrit.nonEmpty) m.koulutusalaKoodiUrit
-                  else
-                    Seq(
-                      koodistoClient.getKoodiUriWithLatestVersion("kansallinenkoulutusluokitus2016koulutusalataso1_01")
+                Some(
+                  m.copy(
+                    isMuokkaajaOphVirkailija = Some(isOphVirkailija),
+                    opintojenLaajuusKoodiUri =
+                      fixedKoodiUriIfNotDefined(m.opintojenLaajuusKoodiUri, "opintojenlaajuus_60"),
```

Comment:
Mun mielestä olis tosi hyvä, että ei enää uusille koulutustyypeille käytettäis näitä `opintojenlaajuus`-koodiston arvoja, vaan laitettais kentät `opintojenLaajuusNumero` ja `opintojenLaajuusyksikkoKoodiUri` erikseen.